Walkthroughs/ImageWhiteout/README.md

```diff
@@ -0,0 +1,215 @@
+#### Build The Image
+Build the image.
+
+```shell
+$ docker buildx build --no-cache -t password-test .
+
+[+] Building 10.0s (10/10) FINISHED                                                                                                  docker:desktop-linux
+ => [internal] load build definition from Dockerfile                                                                                                 0.0s
+ => => transferring dockerfile: 192B                                                                                                                 0.0s
+ => [internal] load .dockerignore                                                                                                                    0.0s
+ => => transferring context: 2B                                                                                                                      0.0s
+ => [internal] load metadata for docker.io/library/ubuntu:latest                                                                                     0.0s
+ => [1/5] FROM docker.io/library/ubuntu                                                                                                              0.0s
+ => [internal] load build context                                                                                                                    0.0s
+ => => transferring context: 74B                                                                                                                     0.0s
+ => CACHED [2/5] WORKDIR /root                                                                                                                       0.0s
+ => [3/5] RUN apt update && apt install -y nodejs                                                                                                    9.5s
+ => [4/5] COPY MySecretPassword .                                                                                                                    0.0s
+ => [5/5] RUN rm -f MySecretPassword                                                                                                                 0.2s
+ => exporting to image                                                                                                                               0.2s
+ => => exporting layers                                                                                                                              0.2s
+ => => writing image sha256:95d8ed919c72b400e96ba2765797e12ae76cef6094f2b34b1d69a1a9be247bac                                                         0.0s
+ => => naming to docker.io/library/password-test                                                                                                     0.0s
+
+View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/0rs3c1htzxrti3znimng1fmn6
+```
+
+#### Inspect the Image
+Let's use the `docker inspect` command to view the contents of our image; note that I am using a `jq` filter to cut 
```

Comment:
It would be a good idea to assume the reader is unfamiliar with `docker inspect` and `jq`. Providing these as inline links in the prose portion of this would add value.